<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulatorGui</class>
 <widget class="QMainWindow" name="SimulatorGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimulatorGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Instruction Memory Table</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2" colspan="2">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>53</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="7" rowspan="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>26</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="10">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Data Memory Table</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="9" colspan="2">
     <widget class="QTableView" name="instructionMemory">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoScrollMargin">
       <number>10</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="10" rowspan="9">
     <widget class="QTableView" name="dataMemory">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="2">
     <widget class="QLabel" name="accumulatorOutputLabel">
      <property name="text">
       <string>Accumulator</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="3">
     <widget class="QTextBrowser" name="accumulator">
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>31</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="4" colspan="3">
     <widget class="QLabel" name="errorOuputLabel">
      <property name="text">
       <string>Standard Output</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>64</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="3" colspan="6">
     <widget class="QTextBrowser" name="standardOutput">
      <property name="maximumSize">
       <size>
        <width>261</width>
        <height>51</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="9">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>43</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="4" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Program Counter</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="4">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>98</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="6" colspan="2">
     <widget class="QLCDNumber" name="programCounter">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="8" colspan="2">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>55</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="6">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>66</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="step">
      <property name="text">
       <string>Step</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="run">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2" colspan="9">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>497</width>
        <height>24</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="8" rowspan="2" colspan="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>49</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2" rowspan="2" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>70</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
